providers/config_default.yaml

```diff
@@ -502,6 +502,7 @@ TKG_HTTP_PROXY: ""
 TKG_HTTPS_PROXY: ""
 TKG_NO_PROXY: ""
 TKG_PROXY_CA_CERT: ""
+TKG_SYSTEM_WIDE_PROXY: false
```

Comment:
maybe TKG_NODE_SYSTEM_WIDE_PROXY

Reply:
Will update that once we got a dev build